WORKSPACE

```diff
@@ -22,7 +22,12 @@ git_repository(
 
 bind(
     name = "protobuf_clib",
-    actual = "@tf//google/protobuf:protobuf",
+    actual = "@tf//google/protobuf:protoc_lib",
+)
+
+bind(
+    name = "protobuf_compiler",
```

Comment:
I agree. I was happy to make it work any way.

But now I find out how to do it right :) See next commit...